raster-foundry/raster-foundry
# Convert BacksplashImage to a trait

## Overview

Paired with @jisantuc on this one.

This PR converts `BacksplashImage` to a trait that takes a parametrized effect type. It also creates a new `BacksplashGeotiff` case class that extends `BacksplashImage`.

### Checklist

- ~Styleguide updated, if necessary~
- ~Swagger specification updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

- Reassemble `backsplace-server`
- Stop memcached service
- Turn `BACKSPLASH_ENABLE_GDAL` to `false` for `backsplash` service in `docker-compose`
- Spin up servers and frontend
- Make sure tiles load for existing and new projects
- Make sure lab still works
